plugins/plugin-compiler.js

```diff
@@ -25,6 +25,11 @@ class WebpackPlugin {
     const file = plugin.generate(data);
 
     await createFile(file, plugin.name, plugin.outputDirectory, plugin.outputLocation);
+
+    //If there is an aditional action to perform
+    if (!!plugin.aditional) {
```

Comment:
what do you think about calling this something like `postcreate`. the idea being its more clear when it's actually occuring, and this could apply to future additional steps like that too

Reply:
I like it, more descriptive. Changing it.